server/common.ts

```diff
@@ -306,6 +306,7 @@ export async function setDefaultSettings() {
 		"confirmationClose": new Date(),
 		"teamsEnabled": true,
 		"qrEnabled": true,
+		"exportKey": crypto.randomBytes(16).toString("hex"),
```

Comment:
best expressed as a configuration option (so you can set an env var or file with this key on multiple instances).
I think we should eventually merge settings + configuration, what do you think @petschekr ?

Reply:
@illegalprime I figured this might be better as a setting, since we wouldn't have to restart the server to change the key